tb_plugin/torch_tb_profiler/plugin.py

```diff
@@ -269,6 +271,15 @@ def trace_route(self, request):
         headers.extend(TorchProfilerPlugin.headers)
         return werkzeug.Response(raw_data, content_type="application/json", headers=headers)
 
+    @wrappers.Request.application
+    def dist_gpu_info_route(self, request):
```

Comment:
the backend/rank/world_size need to be shown in distributed view or in overview?

Reply:
Do you have any suggestion about how to show it? Maybe add it in another pr?